src/burnchains/bitcoin/spv.rs

```diff
@@ -174,11 +177,16 @@ impl SpvClient {
     fn db_instantiate(conn: &mut DBConn) -> Result<(), btc_error> {
         let tx = tx_begin_immediate(conn)?;
 
-        for row_text in SPV_SQL {
+        for row_text in SPV_INITIAL_SCHEMA {
```

Comment:
I think we should just migrate everything to use `execute_batch()` for DB instantiation.  Then this problem will never happen again, no matter how we structure the schema strings.

Reply:
Well, we'll still have to debug bad SQL statements, like a missing `,`, `;` typos, etc.

Reply:
That's fine.  The problem here isn't that we have to debug bad SQL statements; the problem is that SQL statements aren't being executed in the first place.

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/2420/commits/90b1bceb59313d54ce1a3a35a0361ce9f30ca448